example/hello.nf.test

```diff
@@ -6,9 +6,13 @@ nextflow_pipeline {
   test("hello world example should start 4 processes") {
 
     expect {
```

Comment:
Could you please create a separate example for `assertAll`? This file is used by a testcase and it fails now because it expects an exit code 0.